Allow also the height to be adjusted

ale-rt opened the pull request.

reinhardt commented:
I found an issue - when the first card in the first row has double height (8), then the first card in the second row overlaps the large one.

![dashboard-card-overlaps](https://user-images.githubusercontent.com/202439/115676193-a28fa980-a34f-11eb-850d-66ff2caac7b6.png)

ale-rt commented:
I rebased to master after the recent merges and fixed the row_height calculation:
```
row_height = max(height, row_height)
```
that should solve the issue.
